src/frontend/ts/types/urbitChess.ts

```diff
@@ -1,3 +1,5 @@
+import { Key as CgKey } from 'chessground/types'
```

Comment:
Would you kindly please fix the import in `ts/types/chessground.ts` in a similar way